Client/WarFare/UIVarious.cpp

```diff
@@ -699,6 +704,34 @@ void CUIKnights::RefreshButtonHandler(bool blBypassTime)
 	MsgSend_MemberInfoAll();
 }
 
+void CUIKnights::ClanPartyButtonHandler()
+{
+	int iSel = m_pList_CharIDs->GetCurSel();
+	if (iSel == -1)
+		return;
+
+	std::string szID;
+	m_pList_CharIDs->GetString(iSel, szID);
+
+	std::string myID = CGameProcedure::s_pPlayer->IDString();
```

Comment:
2 things:
1. We have no reason to copy this value. Prefer to use a const reference (since we're also not changing it), i.e.: `const std::string& myID = CGameBase::s_pPlayer->IDString();`
2. The other thing is that s_pPlayer is a `CGameBase` member, not `CGameProcedure`. It works because of inheritance, but it's incorrect and misleading.